qiita_db/test/test_metadata_template.py

```diff
@@ -652,6 +775,133 @@ def test_create(self):
                ['2.Sample3', "Value for sample 3"]]
         self.assertEqual(obs, exp)
 
+    def test_create_int_prefix(self):
+        """Creates a new SampleTemplate"""
+        st = SampleTemplate.create(self.metadata_int_pref, self.new_study)
+        # The returned object has the correct id
+        self.assertEqual(st.id, 2)
+
+        # The relevant rows to required_sample_info have been added.
+        obs = self.conn_handler.execute_fetchall(
+            "SELECT * FROM qiita.required_sample_info WHERE study_id=2")
+        # sample_id study_id physical_location has_physical_specimen
+        # has_extracted_data sample_type required_sample_info_status_id
+        # collection_timestamp host_subject_id description
+        exp = [["2.12.Sample1", 2, "location1", True, True, "type1", 1,
```

Comment:
Ok. I think I see your point @wasade 
In the codebase, we are using the aggregated sample IDs as a unique sample id, without having the idea that they are actually aggregated data. So, this goes back to the real question of, how do we solve issue #790?
- Solution 1 (proposed on this PR): add a test on template load to check that the study ids were already present and we don't need to re-add them:
  * PROS: Completely transparent to the user
  * CONS: There is one place on the code base where we actually check that the sample id is `study_id.sample_id`
- Solution 2 (proposed by @wasade): add the ability to download un-prefixed templates
  * PROS: We don't need an implicit check
  * CONS: We still need to trim those study ids at download time. The user will need to know that for meta analyses he needs the prefixed template. The user will need to know that he has to download the un-prefixed version to be able to re-upload that template to QIITA.
- Solution 3: others?

Please, @biocore/qiita-dev , vote, add/remove and propose so we can reach a consensus on this